src/__tests__/depsUtils.test.ts

```diff
@@ -8,6 +8,17 @@ describe("getLicense", () => {
   it("it should return license", () => {
     expect(getLicense(deps({ license: "MIT" }))).toBe("MIT");
     expect(getLicense(deps({ license: { type: "MIT", url: "https://example.com" } }))).toBe("MIT");
+    expect(getLicense(deps({ licenses: [{ type: "MIT", url: "https://example.com" }] }))).toBe("MIT");
+    expect(
+      getLicense(
+        deps({
+          licenses: [
+            { type: "MIT", url: "https://example.com/MIT" },
+            { type: "ISC", url: "https://example.com/ISC" },
+          ],
+        })
+      )
+    ).toBe("(MIT OR ISC)");
```

Comment:
Could you add a test using an actual package.json file to the basic-cases directory under tests/integrations? Since license information is retrieved via npm query, we need to prepare a test that uses it.